legend-engine-xt-persistence-cloud-pure/pom.xml

```diff
@@ -68,6 +69,11 @@
                         <artifactId>legend-engine-xt-persistence-pure</artifactId>
                         <version>${project.version}</version>
                     </dependency>
+                    <dependency>
```

Comment:
Is the legend-engine-xt-relationalStore-pure duplicated in lines 72 and 110 ?

Reply:
No, believe these are needed because they're targeting different plugins.